releases/v20.1.0-rc.1.md

```diff
@@ -0,0 +1,276 @@
+- Fixed a bug where operations on an index that contained a collated string in descending order would fail. [#46570][#46570]
+- Fixed a bug where `SET TIME ZONE` with a string prefixed with `UTC` or `GMT` and time zones that have colons had it's offset inverted the wrong way. [#46510][#46510]
+- CockroachDB no longer incorrectly accounts for some RAM usage when computing aggregate functions. [#46545][#46545]
+- `SHOW INDEXES ... WITH COMMENT` no longer shows duplicate rows for certain tables if indexes are identically named. [#46621][#46621]
+- Fixed an internal error that could happen during planning when a column with a histogram was filtered with a predicate of a different data type. [#46552][#46552]
+- Scans that lock rows (via `FOR UPDATE`) are no longer elided when the results are unused. [#46676][#46676]
+- Fixed a bug (introduced in v20.1.0-beta.3) in the new schema change GC job implementation which would cause the execution of GC jobs to be incorrectly delayed in the presence of other table descriptor updates. [#46691][#46691]
+- Fixed a bug with distinct aggregations on `JSONB` columns. [#46711][#46711]
+- Fixed a rare bug causing the assertion failure "caller violated contract: discovered non-conflicting lock". [#46744][#46744]
+- Ensured that index and table GC happen closer to their GC deadline. [#46743][#46743]
+- Statement diagnostics created through `EXPLAIN ANALYZE (DEBUG)` now show up in the UI page. [#46804][#46804]
+- Benign "outbox next" errors are now only logged when log verbosity is set to 1 or greater. [#46838][#46838]
+- Failed or canceled `IMPORT`s now properly clean up partially imported data. [#46856][#46856]
+- Failed or canceled `RESTORE`s now properly clean up partially imported data. [#46855][#46855]
+- Fixed a rare bug causing errors to be returned for successfully committed transactions. The most common error message was "TransactionStatusError: already committed". [#46848][#46848]
+- The "insecure cluster" indicator is once again displayed at the top right of Admin UI for insecure clusters. [#46865][#46865]
+- Fixed a rare assertion failure that contained the text "existing lock cannot be acquired by different transaction". This assertion was only present in earlier v20.1 releases and not in any earlier releases. [#46896][#46896]
+- Fixed an incorrect query result that could occur when a scalar aggregate was called with a null input. [#46898][#46898]
+- Fixed incorrect result with count(*) when grouping on constant columns. [#46891][#46891]
+- `cockroach demo` now properly cleans up its temporary files if the background license acquisition fails. [#47007][#47007]
+- Tooltips for statement diagnostics are now only shown on hover. [#46995][#46995]
+- Fixed a bug when queries with projections only of `INT2` and/or `INT4` columns were executed via the vectorized engine. [#46977][#46977]
```

Comment:
~Could this also be "of only" (easier to read)?  Or does that change the meaning?